instagram-app/src/App.js

```diff
@@ -9,16 +9,35 @@ class App extends React.Component {
   constructor(){
     super()
       this.state={
-        data:dummyData,
+        data:[],
+        searchPosts:[]
        }
```

Comment:
I would suggest a different name for the `searchPosts` state variable here, since it shares its name with the event method used for search functionality. Perhaps another name (like `filteredPosts`, to provide one suggestion) would be more appropriate here.